libs-scala/contextualized-logging/src/test/suite/scala/com/digitalasset/logging/ContextualizedLoggerIT.scala

```diff
@@ -96,7 +96,7 @@ object ContextualizedLoggerIT {
     val contextualizedLogger = ContextualizedLogger.createFor(logger)
     t(contextualizedLogger)
     output.close()
-    output.toString.split(System.lineSeparator())
+    output.toString.split(System.lineSeparator()).toIndexedSeq
```

Comment:
Maybe another `.view` here to avoid the intermediate collection.

```suggestion
    output.toString.view.split(System.lineSeparator()).toIndexedSeq
```

Reply:
This doesn’t actually compile. `.view` doesn’t have a `split`. Given that this is just a test, I’ll go back to what I had before.